docs/cheatsheet.md

```diff
@@ -0,0 +1,168 @@
+#                           w :     www.aquaq.co.uk                            #
+#                           e : support@aquaq.co.uk                            #
+#                                                                              #
+#                         Running on kdb+ 4 2020.06.18                         #
+#                                                                              #
+#                      TorQ Finance Starter Pack v 1.9.1                       #
+#                                                                              #
+################################################################################
+q)
+```
+
+## Deploying
+
+Deployments should be very simple on Linux [using the installation script](http://aquaqanalytics.github.io/TorQ/InstallGuide/). 
+
+```
+// pull in latest TorQ FSP install script 
+wget https://raw.githubusercontent.com/AquaQAnalytics/TorQ-Finance-Starter-Pack/master/installlatest.sh
+// execute it
+bash installlatest.sh 
+// It will finish with a message like below:
+=============================================================
+INSTALLATION COMPLETE
+=============================================================
+Installation is finished. For a regular installation, run it as follows in the working directory: ./deploy/bin/torq.sh start all
+// you may need to change the value of KDBBASEPORT held in deploy/bin/setenv.sh to avoid conflicts with other TorQ stacks if running on a shared host
+// once done execute start line and check it:
+torquser@homer:/home/torquser/newdeploy$ ./deploy/bin/torq.sh start all
+08:28:02 | Starting discovery1...
+08:28:02 | Starting tickerplant1...
+... snip ...
+08:28:06 | Starting dqe1...
+08:28:06 | Starting dqedb1...
+torquser@homer:/home/torquser/newdeploy$ ./deploy/bin/torq.sh summary
+TIME      |  PROCESS        |  STATUS  |  PID    |  PORT
+08:28:30  |  discovery1     |  up      |  23221  |  43101
+08:28:30  |  tickerplant1   |  up      |  23333  |  43100
+08:28:30  |  rdb1           |  up      |  23443  |  43102
+08:28:30  |  hdb1           |  up      |  23553  |  43103
+... snip ...
+```
+
+## Modifying Existing Installations
+We always try to esnure new versions of TorQ are backwardly compatible. Try to avoid modifying TorQ itself and instead make application specific modifications. The installation script deploys TorQ in a structure which keeps TorQ and the application separate, and this should be adhered to whenever possible. 
+
+To modify the config for a process, do it either in the [config directory](http://aquaqanalytics.github.io/TorQ/gettingstarted/#configuration-loading) or as a command line start up parameter as- almost all config variables that are defined in a config file and exists in a non-root namespace can be overridden from the command line. 
+
+To make a process load additional files, you can:
+
+- append the additional files to the -load parameter on the start line
```

Comment:
might be worth mentioning this can be done in `process.csv` & then `torq.sh` will pick it up?